schema-updater/files_to_be_run.txt

```diff
@@ -2,3 +2,7 @@
 
 update_sqls/create_executed_scripts_tbl.sql
 update_sqls/update_baseentity_constraints.sql
+update_sqls/update_attribute_constraints.sql
+update_sqls/update_baseentity_attribute_constraints.sql
+update_sqls/update_question_constraints.sql
```

Comment:
Do we still need these two scripts to run?

Reply:
Yes. The schema needs to be updated for the infinispan persistence to work.